app/models/podcast_episode.rb

```diff
@@ -42,6 +42,7 @@ class PodcastEpisode < ApplicationRecord
   scope :for_user, lambda { |user|
     joins(:podcast).where(podcasts: { creator_id: user.id })
   }
+  scope :eager_load_serialized_data, -> {}
```

Comment:
There's nothing to eager load for podcast episodes. This is simply for the `BulkIndexWorker`:
```
    def perform(object_class, ids)
      data_hashes = object_class.constantize.
        eager_load_serialized_data.

# etc etc
```